documentation/writing_yaml_plans.md

```diff
@@ -416,6 +416,50 @@ steps:
       message: "The count is ${count}, and twice the count is ${double_count}"
 ```
 
+### Prompt step
+
+The `prompt` step displays a prompt to the user and waits for input.
+
+Prompt steps support the following keys:
+
+| Key | Type | Description | Required |
+| --- | --- | --- | --- |
+| `default` | `String` | The default value to use if the user does not provide input or if standard input (stdin) is not a tty. | |
```

Comment:
This can be `Variant[Data, Target]` right? If that's what the keys of the menu hash accept.

Reply:
Updated to `Array`, `Boolean`, `Hash`, `Number`, `String`